### ISSUE TITLE
shows not an editor command `IlluminationDisable!` 

### ISSUE THREAD

**Issue by arionrefat** [RRethy/vim-illuminate#117]
**Describe the bug**
Not an editor command `IlluminationDisable!` 

This is my minimal init

```lua
local function lsp_highlight_document(client)
	-- Set autocommands conditional on server_capabilities
	local status_ok, illuminate = pcall(require, "illuminate")
	if not status_ok then
		return
	end
	illuminate.on_attach(client)
	-- end
end

M.on_attach = function(client, bufnr)
	if client.name == "tsserver" then
		client.resolved_capabilities.document_formatting = false
	elseif client.name == "sumneko_lua" then
		client.resolved_capabilities.document_formatting = false
	elseif client.name == "gopls" then
		client.resolved_capabilities.document_formatting = false
	end
	lsp_keymaps(bufnr)
	lsp_highlight_document(client)
end
```

**Expected behavior**
It should not display this error

**Screenshots**
![image](https://user-images.githubusercontent.com/37513070/185750673-f0fb7fe4-1e25-4495-927a-8b8ddf6585d1.png)

**Comment by fdelacruz** [comment]
I was just about to report this also.

**Comment by RRethy** [comment]
v2 for the plugin has been released, see [here](https://github.com/RRethy/vim-illuminate/issues/111). You don't need `on_attach` anymore in your config, and these are the provided commands `IlluminatePause`, `IlluminateResume`, `IlluminateToggle`, `IlluminatePauseBuf`, `IlluminateResumeBuf`, `IlluminateToggleBuf`.

The issue you posted was an oversight, if you want to ignore the v2 and continue using `on_attach` things will work as they did before with the fix I pushed.
